modules/ROOT/pages/fcos-projects.adoc

```diff
@@ -2,6 +2,7 @@
 
 This a list of projects that are actively using Fedora CoreOS:
 
+* https://docs.podman.io/en/latest/markdown/podman-machine.1.html[Podman Machine] uses Fedora CoreOS to run containers in a local environment, and also has a `podman machine os apply` command that allows customization.
```

Comment:
Might be worth mentioning here that it's available on non-Linux.

Reply:
Done